<?xml version="1.0"?>
<launch>
    <arg name="odom_topic" default="/odom"/>
    <arg name="flatness_polycoeffs_topic" default="/flatness_polycoeffs"/>
    <arg name="cmd_mode" default="min_jerk"/>
    <node name="controller_node" pkg="uav_geometry_control" type="controller_node"  output="screen">
        <rosparam command="load" file="$(find uav_geometry_control)/config/param.yml"/>
        <param name="odom_topic" value="$(arg odom_topic)"/>
        <param name="flatness_polycoeffs_topic" value="$(arg flatness_polycoeffs_topic)"/>
        <param name="if_debug" value=""/>
        <param name="rate" value="100"/>
        
    </node>
    <node name="fake_odom_sender" pkg="uav_geometry_control" type="fake_odom_sender.py" output="screen">
        <param name="odom_topic" value="$(arg odom_topic)"/>
    </node>
    <node name="fake_cmd_sender" pkg="uav_geometry_control" type="fake_cmd_sender.py"  output="screen">
        <rosparam command="load" file="$(find uav_geometry_control)/config/param.yml"/>
        <param name="odom_topic" value="$(arg odom_topic)"/>
        <param name="flatness_polycoeffs_topic" value="$(arg flatness_polycoeffs_topic)"/>
        <param name="piece_r" value="0.5"/>
    </node>
</launch>